source/declarations.tex

```diff
@@ -735,6 +735,23 @@
 typedef decltype([]{}) C;       // the closure type has no name for linkage purposes
 \end{codeblock}
 \end{example}
```

Comment:
I think if you add a newline after this example, the next sentence will start out on a new line and look better.

Reply:
Fixed.